### DISCUSSION TITLE
How to register flows without a label?

### DISCUSSION THREAD

**Discussion by sti0** [PrefectHQ/prefect#4045] [Q&A] [answered]
How could I register a new flow without a label? At the moment the hostname gets added every time, even if I specify a label name.

Tried:

`flow.register(project_name="myproject", idempotency_key=flow.serialized_hash(), labels=None)` -> Labels: ['MYCOMPUTERNAME']
`flow.register(project_name="myproject", idempotency_key=flow.serialized_hash(), labels=[])` -> Labels: ['MYCOMPUTERNAME']
`flow.register(project_name="myproject", idempotency_key=flow.serialized_hash(), labels=["dev"])` -> Labels: ['MYCOMPUTERNAME', 'dev']

**Comment by joshmeek** [comment, marked as answer]
Hi @sti0 try `flow.register(add_default_labels=False)`

**Reply by sti0** [reply]
Thank you. Works like a charm
